### ISSUE TITLE
Fix Excel Report exporter

### ISSUE THREAD

**Issue by jmcameron** [Third-Culture-Software/bhima#6313]
In the server, when we generate an 'xlsxReport', it appears to work, but the resulting seems to be corrupted.  When you open the resulting Excel file, it complains:
![image](https://user-images.githubusercontent.com/62145/150408709-b1bf1719-4c14-4cc7-be19-4d1790141565.png)

If you say yes, the result seem to be okay.   Not sure what is going on.

However, the 'xlsx' export works without problems.
- See Stock > Stock Lots > Menu > Download as Excel

**To reproduce the problem:**
- Stock > Articles in Stock
  - Verify that the export type is 'xlsxReport' (line 312 client/src/modules/stock/inventories/registry.js)
  - download:  Stock > Articles in Stock > Menu > Download as Excel
  - Open the downloaded file and see the error message

**Comment by jniles** [comment]
Yeah, this is because the XLS export isn't actually an XLS export - it is an HTML file renamed to XLSX.  We probably should figure out something better on the long run.

One issues is that the `xlsx` library we use doesn't support styling cells - I believe it is a paid feature.  `excel4node` does, but it doesn't appear to be maintained anymore.  It would be worth us reflecting on it and choosing a single library for our exports that supports all our needs.

**Comment by jmcameron** [comment]
See related Issue https://github.com/IMA-WorldHealth/bhima/issues/6342
